pages/express-relay/contract-addresses/svm.mdx

```diff
@@ -3,26 +3,75 @@ import AddressSvmTable from "../../../components/AddressSvmTable";
 
 Express Relay is currently deployed on the following SVM environments:
 
-<Tabs items={['Solana Mainnet-Beta Testing']}>
+<Tabs items={['Prod (Solana Mainnet-Beta)', 'Testing (Solana Mainnet-Beta)']}>
+<Tabs.Tab>
+Auction Server endpoint: https://pyth-express-relay-mainnet.asymmetric.re/
+
+### Prod
+
+**Network Details:**
+
+<AddressSvmTable
+  explorer={"https://solscan.io/account/$ADDRESS"}
+  entries={[
+    {
+      name: "Express Relay Program",
+      value: "PytERJFhAKuNNuaiXkApLfWzwNwSNDACpigT3LwQfou",
+      link: true,
+    },
+    {
+      name: "Limo Limit Order Program",
+      value: "LiMoM9rMhrdYrfzUCxQppvxCSG1FcrUK9G8uLq4A1GF",
+      link: true,
+    },
+    {
+      name: "Chain Id",
+      value: "solana",
+      link: false,
+    },
+    {
+      name: "Public RPC",
+      value: "https://api.mainnet-beta.solana.com",
+      link: false,
+    },
+    {
+      name: "Limo Global Order Config",
+      value: "prod7Mog8r56sn5dPwze8nVdf94veEawdxNNqd5v7A1",
+      link: true,
+    },
+    {
+      name: "Frontend",
+      value: "https://swap.kamino.finance",
+      link: false,
```

Comment:
```suggestion
      link: true,
```

Reply:
this table needs to be refactored a bit if we want to use it for non-explorer links. A future TODO